dtests/src/test/scala/io/snappydata/hydra/ct/CTTestUtil.scala

```diff
@@ -106,11 +107,25 @@ object CTTestUtil {
         + "')")
   }
 
+  def createColumnTablesWithKeyColumns(snc: SnappyContext, redundancy: String): Unit = {
+    snc.sql(CTQueries.orders_details_create_ddl + " using column options(redundancy '" +
+        redundancy + "' , key_columns 'SINGLE_ORDER_DID')")
+    snc.sql(CTQueries.exec_details_create_ddl + " using column options(redundancy '" + redundancy
+        + "', key_columns 'EXEC_DID')")
+  }
+
   def createPersistColumnTables(snc: SnappyContext, persistenceMode: String): Unit = {
     snc.sql(CTQueries.orders_details_create_ddl + " using column options(PERSISTENT '" +
         persistenceMode + "')")
     snc.sql(CTQueries.exec_details_create_ddl + " using column options(PERSISTENT '" +
-        persistenceMode + "')")
+        persistenceMode + "', key_columns 'EXEC_DID')")
```

Comment:
I think this is accidentally added here. It seems you have written a new method for create column tables with key column.